README.md

```diff
@@ -123,6 +123,15 @@ You can also attach multiple listeners to a given message, hence allowing to do
     ]
 ```
 
+#### Registering WorkerMiddleware
+You should register the `WorkerMiddleware` in your router to respond the "/internal/worker" path, this middleware consumes the messages sent by Elastic Beanstalk worker environment and routes to the mapped listener. For example, in Zend Expressive:
```

Comment:
After ""/internal/worker" path", start a new sentence ("This middleware ..."). It's a bit long here.